mappings/net/minecraft/potion/PotionUtil.mapping

```diff
@@ -8,6 +8,10 @@ CLASS net/minecraft/unmapped/C_lyeejpyw net/minecraft/potion/PotionUtil
 		ARG 0 potion
 	METHOD m_cxismorv getColor (Ljava/util/Collection;)I
 		ARG 0 effects
+	METHOD m_dayagjta appendTooltip (Ljava/util/List;Ljava/util/List;F)V
```

Comment:
Toes stepped by #336, where `appendDetailsToTooltip` was used; hm, i feel like the more verbose one might be better but yeah, am not 100% sure

Reply:
just going to throw out tooltip stuff here